apps/bulk-uploader/src/components/BulkUploader.vue

```diff
@@ -0,0 +1,205 @@
+						</template>
+						<template #file-upload-instructions>
+							<p class="text-color-gray-medium-dark">
+								If the bulk upload contains base fields which are file
+								attachments, you can upload a zip file with the attachments.
+								Ensure that the base field values are relative paths to the
+								files from the root of the zip file.
+							</p>
+						</template>
+					</FileUploadInput>
+				</template>
+			</PanelSection>
+			<PanelSection>
+				<template #header>
+					<h3>Associated Entities</h3>
+				</template>
+				<template #content>
+					<SelectInput
+						:model-value="props.sourceId"
+						:options="
+							props.sources?.entries.map((source) => ({
+								label: source.label,
+								value: source.id.toString(),
+							}))
+						"
+						@update:model-value="
+							(value: string | null | undefined) =>
+								emit('update:source-id', value ?? null)
+						"
+					>
+						<template #select-input-header>
+							<h4>Source</h4>
+						</template>
+						<template #select-input-instructions>
+							<p class="text-color-gray-medium-dark">
+								If blank, a <span>sourceId</span> column must be included in the
+								upload.
+							</p>
+						</template>
+					</SelectInput>
+					<SelectInput :options="[]" :disabled="true">
+						<template #select-input-header>
+							<h4>Data Provider</h4>
+						</template>
+						<template #select-input-instructions>
+							<p class="text-color-gray-medium-dark">
+								If blank, a
+								<span>dataProviderShortCode</span> column must be included in
+								the upload.
+							</p>
+						</template>
+					</SelectInput>
+					<SelectInput
+						:model-value="props.funderShortCode"
+						:options="
+							props.funders?.entries.map((funder) => ({
+								label: funder.name,
+								value: funder.shortCode,
+							}))
+						"
+						@update:model-value="
+							(value: string | null | undefined) =>
+								emit('update:funder-short-code', value ?? null)
+						"
+					>
+						<template #select-input-header>
+							<h4>Funder</h4>
+						</template>
+						<template #select-input-instructions>
+							<p class="text-color-gray-medium-dark">
+								If blank, a
+								<span>funderShortCode</span> column must be included in the
+								upload.
+							</p>
+						</template>
+					</SelectInput>
+					<SelectInput :disabled="true">
+						<template #select-input-header>
+							<h4>Changemaker</h4>
+						</template>
+						<template #select-input-instructions>
+							<p class="text-color-gray-medium-dark">
+								If blank, a
+								<span>changemakerId</span> column must be included in the
+								upload.
+							</p>
+						</template>
+					</SelectInput>
+					<SelectInput :disabled="true">
+						<template #select-input-header>
+							<h4>Application Form</h4>
+						</template>
+						<template #select-input-instructions>
+							<p class="text-color-gray-medium-dark">
+								If blank, an
```

Comment:
"If no Application Form is specified here, then the uploaded CSV must include an `applicationFormId` column."

...plus all the earlier questions, adjusted as needed :-).

Reply:
This is out of date too -- application form can't be provided by the bulk uploader because we only support a single bulk upload flow for this MVP, which is "the application form + opportunity are created for the purposes of housing this bulk upload's proposal data"